pkg/agent/cniserver/ipam/antrea_ipam.go

```diff
@@ -115,21 +115,22 @@ func (d *AntreaIPAM) Add(args *invoke.Args, k8sArgs *argtypes.K8sArgs, networkCo
 	owner := getAllocationOwner(args, k8sArgs, reservedOwner)
 	var ip net.IP
 	var subnetInfo crdv1a2.SubnetInfo
+	var vlanTag *uint32
 	if reservedOwner != nil {
-		ip, subnetInfo, err = allocator.AllocateReservedOrNext(crdv1a2.IPAddressPhaseAllocated, owner)
+		ip, subnetInfo, vlanTag, err = allocator.AllocateReservedOrNext(crdv1a2.IPAddressPhaseAllocated, owner)
```

Comment:
A general suggestion - why not use "VLAN ID / vlanID", instead of "VLAN Tag" for VLAN ID?

Reply:
Fixed.